pep-0683.rst

```diff
@@ -503,11 +503,11 @@ to do its best effort of deallocating these instances normally.  Most
 of the module deallocation will now be handled by
 ``pylifecycle.c:finalize_modules()`` which cleans up the remaining
 modules as best as we can.  It will change which modules are available
-during __del__ but that's already defined as undefined behavior by the
-docs.  Optionally, we could do some topological disorder to guarantee
+during ``__del__``, but that's already defined as undefined behavior by the
+docs.  Optionally, we could do some topological sorting to guarantee
```

Comment:
I don't think "topological disorder" is a term outside solid-state physics, but might be wrong.

Reply:
Oops, that was a typo on my end! I mostly was referring to a topological 'destruction' not 'disorder' under the guarantee that we have some sort of topological sorting.